lib/containerize_task.rb

```diff
@@ -20,5 +20,15 @@ def containerize(description, dir)
     mapper = WorkloadMapper.new
     workloads = mapper.identify_workloads(description)
     mapper.write_compose_file(workloads, dir)
+    if workloads.empty?
+      Machinery::Ui.puts "No workloads detected."
+    else
+      workloads.each do |workload|
+        Machinery::Ui.puts "Detected workload '#{workload[0]}'."
+      end
+      Machinery::Ui.print "\nFound #{workloads.count}" \
+        " #{Machinery::pluralize(workloads.count, "workload")}."
```

Comment:
I don't think this is helpful output. The number of workloads will always be pretty small.